src/FlightMap/Widgets/QGCInstrumentWidget.qml

```diff
@@ -18,6 +18,10 @@ import QGroundControl.FlightMap     1.0
 import QGroundControl.FlightDisplay 1.0
 import QGroundControl.Palette       1.0
 
+import QGroundControl.Vehicle       1.0
+import QtQuick.Window               2.2
+import MAVLink                      1.0
+
```

Comment:
This breaks the imports list style of the file, which becomes messy in the long run. Merge into import blocks above

Reply:
New widget follows block structure (if I understood the comment correctly).